README.md

```diff
@@ -1,68 +1,44 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Quiz web app
 
-## Available Scripts
+## Description:
+This is a quiz web app which helps users to check their knowledge of CYF study material.
 
-In the project directory, you can run:
+## Installation: 
+1. Clone files into your local machine using - `git clone https://github.com/CodeYourFuture/project-quiz.git`.
+2. Run `npm install` in your local terminal.
+3. Run `npm start` to run the app or open `http://localhost:3000` to view it in the browser.
 
-### `yarn start`
+## Table of Quizzes: 
+    1.HTML.
+    2.CSS.
+    3.JavaScript.
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Running the tests:
+    TBC.
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+## Tech Used / Dependencies:
+ React.
+ HTML.
+ CSS.
+ Node.
 
-### `yarn test`
+## Feature requests:
+When it comes to creating feature requests, it must be discussed and approved by the team members, therefore please provide as much info as possible to be accepted.
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Pull requests:
 
-### `yarn build`
+For contributors team :
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
+    1. Clone or pull the latest master repo and create a branch.
+    2. git checkout -b my-new-feature-branch.
+    3. make code changes.
+    4. git add and git commit.
+    5. git push origin my-new-feature-branch
 
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+    In regards to any addition or changes to a feature, make sure to update it in
+    the README.md.
```

Comment:
These lines should not be indented. Only sub-lists should be indented in markdown

Reply:
It looks like you missed removing the indents in steps 1. to 5. above too. Please remember to check the rest of the file to make everything consistent

Reply:
I see now, I will fix it.